### PR TITLE
Display all the available regions

### LINKED ISSUE #12375: Display all available regions in Data Observatory
Extracted from #12370.

mandatory-region

The "region" dropdown should display all available regions in our DO database instead of showing those regions where the "data belongs".

Check [this comment](https://github.com/CartoDB/cartodb/pull/12370#issuecomment-310316790) about implementation suggestion.

@CartoDB/builder-backend @ethervoid

### PR DESCRIPTION
**Pull request by ethervoid**
Fixes #12375 

Remove the extent in the regions query for DO to get all the available regions without filtering

### CODE REVIEW

File: lib/assets/core/javascripts/cartodb3/data/data-observatory/regions-collection.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 var BaseCollection = require('./data-observatory-base-collection');
 var BaseModel = require('../../components/custom-list/custom-list-item-model');
 
-var REGIONS_QUERY = "SELECT count(*) num_measurements, tag.key region_id, tag.value region_name FROM (SELECT * FROM OBS_GetAvailableNumerators((SELECT ST_SetSRID(ST_Extent(the_geom), 4326) FROM ({{{ query }}}) q)) WHERE jsonb_pretty(numer_tags) LIKE '%subsection/%') numers,  Jsonb_Each(numers.numer_tags) tag WHERE tag.key like 'section%' GROUP BY tag.key, tag.value";
+var REGIONS_QUERY = "SELECT count(*) num_measurements, tag.key region_id, tag.value region_name FROM (SELECT * FROM OBS_GetAvailableNumerators() WHERE jsonb_pretty(numer_tags) LIKE '%subsection/%') numers,  Jsonb_Each(numers.numer_tags) tag WHERE tag.key like 'section%' GROUP BY tag.key, tag.value";
```
Comment (nobuti) [inline lib/assets/core/javascripts/cartodb3/data/data-observatory/regions-collection.js]: `order by region_name` should be nice.

Comment (nobuti) [inline lib/assets/core/javascripts/cartodb3/data/data-observatory/regions-collection.js]: or num_measurements, not sure

**Comment by nobuti** [comment]
@kevin-reilly now that we are going to show all regions, should we sort the results in any way? alphabetically or by the count of measurements?

File: lib/assets/core/javascripts/cartodb3/data/data-observatory/regions-collection.js (hunk 1 above)
Comment (ethervoid) [inline lib/assets/core/javascripts/cartodb3/data/data-observatory/regions-collection.js]: Ok let's wait for @kevin-reilly answer

**Comment by ethervoid** [comment]
We could order by region name and then change if needed

**Comment by kevin-reilly** [comment]
Alphabetical is fine for now.

--
*Kevin Reilly*
SVP Product
*CARTO <https://carto.com/> | kevin@carto.com <kevin@carto.com>*
(917) 375-2168

**Pull request merged**